dotnetv3/Lambda/SetuplambdaRoleExample/SetuplambdaRoleExample/SetupLambdaRole.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+namespace SetuplambdaRoleExample
+{
+    using System;
+    using System.Threading.Tasks;
+    using Amazon.IdentityManagement;
+    using Amazon.IdentityManagement.Model;
+
+    /// <summary>
+    /// Creates an Amazon Identity Management Role to attach to an Amazon
+    /// Lambda Function that will management an Amazon Simple Storage Service
+    /// (Amazon S3) Bucket. The example was created using AWS SDK for .NET
+    /// version 3.7 and .NET Core 5.0.
+    /// </summary>
+    public class SetupLambdaRole
+    {
+        /// <summary>
+        /// Defines the policy for the IAM Role and then creates the role.
+        /// </summary>
+        static async Task Main()
+        {
+            // Policy that allows reading and writing to a specific Amazon S3
+            // Bucket. This role will allow managing the bucket as well as
+            // working with the objects in that bucket.
+            string s3ManagementPolicy = "{" +
+                "   \"Version\": \"2012-10-17\"," +
+                "	\"Statement\" : [{" +
+                    "   \"Sid\": \"ListObjectsInBucket\"," +
+                    "	\"Effect\" : \"Allow\"," +
+                    "   \"Action\" : [\"s3: ListBucket\"]," +
+                    "	\"Resource\" :[\"arn:aws:s3:::doc-example-bucket/*\"]" +
+                "}," +
+                    "   \"Sid\": \"AllObjectActions\"," +
+                    "	\"Effect\" : \"Allow\"," +
+                    "   \"Action\" : [\"s3:*Object*\"]," +
+                    "	\"Resource\" :[\"arn:aws:s3:::doc-example-bucket/*\"]" +
+                "}]" +
+            "}";
+
+            string roleName = "S3ManagementRole";
+
+            // Create the Amazon Identity Management client object.
```

Comment:
I think you can just use "IAM" here as long as you've called out the long name previously.

Reply:
You're right. Fixed